public/app/features/dashboard/dashgrid/DashboardPanel.tsx

```diff
@@ -122,13 +125,28 @@ export class DashboardPanelUnconnected extends PureComponent<Props, State> {
       return null;
     }
 
-    const panelWrapperClass = classNames({
-      'panel-wrapper': true,
-      'panel-wrapper--view': isViewing,
-    });
-
-    return <div className={panelWrapperClass}>{this.renderPanel(plugin)}</div>;
+    return (
+      <div
+        className={isViewing === true ? classNames(styles.panelWrapper, styles.panelWrapperView) : styles.panelWrapper}
+      >
+        {this.renderPanel(plugin)}
+      </div>
+    );
   }
 }
 
+export const getStyles = stylesFactory((theme: GrafanaTheme) => {
+  return {
+    panelWrapper: css`
+      height: 100%;
+      position: relative;
+    `,
+    panelWrapperView: css`
+      flex: 1 1 0;
+      height: 90%;
+    `,
+  };
+});
+export const DashboardPanelUnconnected = withTheme(UnthemedDashboardPanelUnconnected);
+DashboardPanelUnconnected.displayName = 'UnthemedDashboardPanelUnconnected';
```

Comment:
This getStyles does not actually need the theme so you can remove the withTheme bit

Reply:
Alrighty! 👍